language-server/src/server.js

```diff
@@ -72,6 +78,9 @@ connection.onInitialize(({ capabilities, workspaceFolders }) => {
 });
 
 connection.onInitialized(async () => {
+  if (hasConfigurationCapability) {
+    connection.client.register(DidChangeConfigurationNotification.type, undefined);
+  }
```

Comment:
Code Style: I generally like to have whitespace after a block.

```suggestion
  if (hasConfigurationCapability) {
    connection.client.register(DidChangeConfigurationNotification.type, undefined);
  }
  
```

Reply:
Will keep it in mind from next time and double check it for now for all the places this might have occurred.